msat/include/msat_solver.h

```diff
@@ -169,6 +168,10 @@ class MsatSolver : public AbsSmtSolver
   size_t num_assump_clauses_;  ///< counts how many assumption clauses are added
   size_t max_assump_clauses_;  ///< number of assumption clauses before clearing
                                ///< them
+  bool last_query_assuming;    ///< set to true if last query was
+                             ///< check-sat-assuming used to respect
+                             ///< get_unsat_assumptions interface (will complain
+                             ///< if not called after check-sat-assuming)
```

Comment:
I don’t really understand the comments here. Could you try to clarify?

Reply:
Oops, this is partially because of formatting. I keep forgetting to check this. It makes slightly more sense with new lines, but clang-format seems to join them. I improved the wording and spacing with fabffb677ced3bb0aca1130e3bbbb2a2b41d889f.